src/OctoshiftCLI.Tests/gei/Commands/ReclaimMannequinCommandTests.cs

```diff
@@ -0,0 +1,225 @@
+            var mannequinUser = "mona";
+            var mannequinUserId = Guid.NewGuid().ToString();
+            var targetUser = "mona_emu";
+            var targetUserId = Guid.NewGuid().ToString();
+            var targetGithubPat = Guid.NewGuid().ToString();
+
+            var mannequinResponse = new Mannequin
+            {
+                Id = mannequinUserId,
+                Login = "mona"
+            };
+
+            var mockGithub = new Mock<GithubApi>(null, null);
+            mockGithub.Setup(x => x.GetOrganizationId(githubOrg).Result).Returns(githubOrgId);
+            mockGithub.Setup(x => x.GetMannequin(githubOrgId, mannequinUser).Result).Returns(mannequinResponse);
+            mockGithub.Setup(x => x.GetUserId(targetUser).Result).Returns(targetUserId);
+            mockGithub.Setup(x => x.ReclaimMannequin(githubOrgId, mannequinUserId, targetUserId).Result).Returns(true);
+
+
+            var environmentVariableProviderMock = new Mock<EnvironmentVariableProvider>(null);
+            environmentVariableProviderMock.Setup(m => m.TargetGithubPersonalAccessToken()).Returns(targetGithubPat);
+
+            var mockGithubApiFactory = new Mock<ITargetGithubApiFactory>();
+            mockGithubApiFactory.Setup(m => m.Create(TARGET_API_URL)).Returns(mockGithub.Object);
+
+            var command = new ReclaimMannequinCommand(new Mock<OctoLogger>().Object, mockGithubApiFactory.Object, environmentVariableProviderMock.Object);
+            var reclaimed = await command.Invoke(githubOrg, mannequinUser, targetUser, false);
+
+            mockGithub.Verify(x => x.ReclaimMannequin(githubOrgId, mannequinUserId, targetUserId));
+            reclaimed.Should().BeTrue();
+        }
+
+        [Fact]
+        public async Task AlreadyMapped_No_Reclaim()
+        {
+            var githubOrg = "FooOrg";
+            var githubOrgId = Guid.NewGuid().ToString();
+            var mannequinUser = "mona";
+            var mannequinUserId = Guid.NewGuid().ToString();
+            var targetUser = "mona_emu";
+            var targetUserId = Guid.NewGuid().ToString();
+            var targetGithubPat = Guid.NewGuid().ToString();
+
+            var mannequinResponse = new Mannequin
+            {
+                Id = mannequinUserId,
+                Login = "mona",
+                MappedUser = new Claimant
+                {
+                    Id = "claimantId",
+                    Login = "claimant"
+                }
+            };
+
+            var mockGithub = new Mock<GithubApi>(null, null);
+            mockGithub.Setup(x => x.GetOrganizationId(githubOrg).Result).Returns(githubOrgId);
+            mockGithub.Setup(x => x.GetMannequin(githubOrgId, mannequinUser).Result).Returns(mannequinResponse);
+            mockGithub.Setup(x => x.GetUserId(targetUser).Result).Returns(targetUserId);
+            mockGithub.Setup(x => x.ReclaimMannequin(githubOrgId, mannequinUserId, targetUserId).Result).Returns(true);
```

Comment:
is this necessary? below you assert this is never called (same with above GetUserId() setup)

Reply:
fixed [dbb2ee9](https://github.com/github/gh-gei/pull/271/commits/dbb2ee916140516399a5af4b3e3e416f63bc96fc)